sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1391,6 +1394,16 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setMethod(method).build();
     }
 
+    /**
+     * Set the project the BigQuery load job will be initiated from. This is only applicable when
+     * the write method is set to {@link Method#FILE_LOADS}. If omitted, the project of the
+     * destination table is used.
+     */
+    public Write<T> withLoadJobProjectId(String loadJobProjectId) {
```

Comment:
Could you add a ValueProvider version too?

Reply:
Done